terracumber_config/tf_files/SUSEManager-5.0-build-validation-paygo-AWS.tf

```diff
@@ -0,0 +1,501 @@
+    key_file          = var.KEY_FILE
+  }
+}
+
+module "mirror" {
+  source = "./modules/mirror"
+  base_configuration = module.base.configuration
+  disable_cron = true
+  provider_settings = {
+    public_instance = true
+  }
+  image = "opensuse155o"
+}
+
+module "server" {
+  source                     = "./modules/server"
+  base_configuration = merge(module.base.configuration,
+    {
+      mirror = null
+    })
+  name                       = "server"
+  product_version            = "5.0-paygo"
+  image                      = var.SERVER_AMI != "" ? var.SERVER_AMI : "suma-server-50-${var.ARCHITECTURE}-ltd-paygo"
+  main_disk_size             = 200
+  repository_disk_size       = 1500
+  database_disk_size         = 0
+
+  auto_accept                    = false
+  monitored                      = false
+  disable_firewall               = false
+  allow_postgres_connections     = false
+  skip_changelog_import          = false
+  create_first_user              = false
+  mgr_sync_autologin             = false
+  create_sample_channel          = false
+  create_sample_activation_key   = false
+  create_sample_bootstrap_script = false
+  publish_private_ssl_key        = false
+  use_os_released_updates        = true
+  disable_download_tokens        = false
+  large_deployment               = true
+  provision                      = false
+  ssh_key_path            = "./salt/controller/id_rsa.pub"
+  provider_settings = {
+    instance_type = var.ARCHITECTURE == "x86_64" ? "m6a.xlarge" : "m6g.xlarge"
+  }
+
+  //server_additional_repos
+
+}
+
+module "proxy" {
+  source                    = "./modules/proxy_containerized"
+  base_configuration        = module.base.configuration
+  server_configuration      = module.server.configuration
+  name                      = "proxy"
+  proxy_registration_code   = var.PROXY_REGISTRATION_CODE
+  image                     = var.PROXY_AMI != "" ? var.PROXY_AMI : "suma-proxy-50-byos"
+  provision                 = false
+
+
+  auto_configure            = false
+  use_os_released_updates   = true
+  ssh_key_path              = "./salt/controller/id_rsa.pub"
+  provider_settings         = {
+    instance_type = "c6i.large"
+  }
+
+  //proxy_additional_repos
+}
+
+module "sles12sp5_paygo_minion" {
+  source             = "./modules/minion"
+  base_configuration = module.base.configuration
+  product_version    = "paygo"
+  name               = "sles12sp5-paygo-minion"
+  image              = "sles12sp5-paygo"
+  provider_settings = {
+    instance_type = "t3a.medium"
+  }
+  server_configuration = module.server.configuration
+  auto_connect_to_master  = false
+  use_os_released_updates = false
+  ssh_key_path            = "./salt/controller/id_rsa.pub"
+  additional_packages = [ "python-instance-billing-flavor-check" ]
+}
+
+module "sles15sp5_paygo_minion" {
+  source             = "./modules/minion"
+  base_configuration = module.base.configuration
+  product_version    = "paygo"
```

Comment:
That's a valid product_version?

Reply:
There are few more like those.

Reply:
yes, it's to trigger specific tests in the testsuite